Detections/3pNetworkDetetions/SucessfullSiginFromPhingLink.yaml

```diff
@@ -0,0 +1,131 @@
+    "Email messages containing malicious URL removed after delivery",
+    "Email reported by user as malware or phish",
+    "Phish delivered due to an ETR override",
+    "Phish not zapped because ZAP is disabled"]);
+    SecurityAlert | where AlertName in~ (Alert_List)
+    //Findling Alerts which has the URL
+    | where Entities has "url"
+    //extracting Entities 
+    | extend Entities = parse_json(Entities)
+    | mv-apply Entity = Entities on
+        (
+        where Entity.Type == 'account'
+        | extend EntityUPN = iff(isempty(Entity.UserPrincipalName), tostring(strcat(Entity.Name, "@", tostring (Entity.UPNSuffix[0]))), tostring(Entity.UserPrincipalName))
+        )
+    | mv-apply Entity = Entities on
+        (
+        where Entity.Type == 'ip'
+        | extend EntityIp = tostring(Entity.Address)
+        )
+    | mv-apply Entity = Entities on
+        (
+        where Entity.Type == 'host'
+        | extend EntityHost = tostring(Entity.HostName)
+        )
+    | mv-apply Entity = Entities on
+        (
+        where Entity.Type == 'url'
+        | extend EntityUrl = tostring(Entity.Url)
+        )
+    | where not(ipv4_is_private(EntityIp))
+    | summarize
+        AccountUpn=take_any(EntityUPN),
+        IPAddress=take_any(EntityIp),
+        HostName=take_any(EntityHost),
+        Url=take_any(EntityUrl),
+        AlertTime= min(TimeGenerated),
+        make_set(SystemAlertId)
+        by ProductName, AlertName
+    // matching with 3rd party network logs and 3p Alerts 
+    | join kind= inner (CommonSecurityLog
+        | where DeviceVendor has_any  ("Palo Alto Networks", "Fortinet", "Check Point", "Zscaler")
+        | project
+            3plogTime=TimeGenerated,
+            DeviceVendor,
+            DeviceProduct,
+            Activity,
+            DestinationHostName,
+            DestinationIP,
+            RequestURL,
+            MaliciousIP,
+            SourceUserName,
+            IndicatorThreatType,
+            ThreatSeverity,
+            ThreatConfidence,
+            SourceUserID,
+            SourceHostName)
+        on $left.Url == $right.RequestURL
+    // matching successful Login from suspicious IP 
+    | join kind=inner (SigninLogs
+        //filtering the Successful Login
+        | where ResultType == 0
+        | project
+            IPAddress,
+            SourceSystem,
+            SigniningTime= TimeGenerated,
+            OperationName,
+            ResultType,
+            ResultDescription,
+            AlternateSignInName,
+            AppDisplayName,
+            AuthenticationRequirement,
+            ClientAppUsed,
+            RiskState,
+            RiskLevelDuringSignIn,
+            UserPrincipalName)
+        on $left.DestinationIP == $right.IPAddress and $left.SourceUserName == $right.UserPrincipalName  
+    | where SigniningTime between ((AlertTime - 6h) .. (AlertTime + 6h)) and 3plogTime between ((AlertTime - 6h) .. (AlertTime + 6h))   
```

Comment:
Why did you have such different time frames in this query as opposed to the last one?